### ISSUE TITLE
figcaption always renders the first image alt and ignores followings

### ISSUE THREAD

**Issue by kinopyo** [arve0/markdown-it-implicit-figures#26]
Hi, thanks for the charming plugin! 🌟 I'm just about to use it, and bumped into an issue.

Say we have the markdown like this:

```html
![alt](img.png)

![this caption won't be rendered, the 'alt' above is used](fig.png)
```

It'll be rendered as below, note the second `<figcation>`.

```html
<figure>
  <img src="img.png" alt="alt">
  <figcaption>alt</figcaption>
</figure>

<figure>
  <img src="fig.png" alt="this caption won't be rendered, the 'alt' above is used">
  <figcaption>alt</figcaption> <!-- 👈HERE the caption is taken the first image's alt -->
</figure>
```

Demo: https://jsfiddle.net/00Lgnp3x/

Haven't looked into the code yet, maybe would be an easy fix, will first report here to see if it's a known issue or not 🙂. Thanks in advance!

**Comment by arve0** [comment]
Hi! Thanks for reporting. The issue is not known to me. Feel free to poke around in the code.

Hint: Take a look at the token stream at markdown-it.github.io to easier grasp the code.

**Comment by arve0** [comment]
Fix released in `v0.7.1`.

**Comment by kinopyo** [comment]
@arve0 Wow sweet! Big thanks! 👏 👍
